carbonmark/components/pages/Project/Retire/Pool/RetireInputs.tsx

```diff
@@ -86,6 +86,30 @@ const validations = (
       },
     },
   },
+  "bank-transfer": {
+    quantity: {
+      min: {
+        // @todo - makka - confirm with atmos/brice what the min quantity is
```

Comment:
@jabby09 is there a minimum quantity for bank transfers? I've currently set this to the same as credit card.